Include/internal/pycore_fileutils.h

```diff
@@ -251,6 +251,14 @@ extern int _Py_add_relfile(wchar_t *dirname,
 extern size_t _Py_find_basename(const wchar_t *filename);
 PyAPI_FUNC(wchar_t *) _Py_normpath(wchar_t *path, Py_ssize_t size);
 
+// The Windows Games API family does not provide these functions
+// so provide our own implementations. Remove them in case they get added
+// to the Games API family
+#if defined(MS_WINDOWS) && !defined(MS_WINDOWS_APP) && !defined(MS_WINDOWS_SYSTEM)
```

Comment:
ISTM we can specify this positively for `GAMES`, with the exception of `DESKTOP` games.

```suggestion
#if defined(MS_WINDOWS_GAMES) && !defined(MS_WINDOWS_DESKTOP)
```

Otherwise the check needs another term. Per the discussion with Steve, we shouldn't assume that the reader knows that `APP` is true if `DESKTOP` is true. So it becomes `#if defined(MS_WINDOWS) && !defined(MS_WINDOWS_APP) && !defined(MS_WINDOWS_DESKTOP) && !defined(MS_WINDOWS_SYSTEM)`.